### PR TITLE
feat(appication-system-api): Opening for acess over X-Road

### PR DESCRIPTION
**Pull request by pshomov**
# ...

Attach a link to issue if relevant

## What

Specify what you're trying to achieve

## Why

Specify why you need to achieve this

## Screenshots / Gifs

Attach Screenshots / Gifs to help reviewers understand the scope of the pull request

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

**Review by arnorhs — APPROVED** [review]

**Review by jGeir — APPROVED** [review]

**Review by sindrig — APPROVED** [review]
LGTM

File: apps/application-system/api/infra/application-system-api.ts [hunk 1]
```diff
@@ -194,4 +194,15 @@ export const serviceSetup = (services: {
       limits: { cpu: '400m', memory: '512Mi' },
       requests: { cpu: '100m', memory: '256Mi' },
     })
-    .grantNamespaces('nginx-ingress-external', 'islandis')
```
Comment (AndriMar) [inline apps/application-system/api/infra/application-system-api.ts:197]: I would think we would need to grant both external and internal nginx ingress.

**Review by AndriMar — COMMENTED** [review]
I would think we would need to grant both external and internal nginx ingress.?

**Review by AndriMar — APPROVED** [review]

File: apps/application-system/api/infra/application-system-api.ts (hunk 1 above)
Comment (AndriMar) [reply apps/application-system/api/infra/application-system-api.ts:196]: Got confirmation from  @arnorhs that no external traffic will be to this application.

**Pull request merged**